.gitignore

```diff
@@ -5,4 +5,5 @@
 output/*
 aur-pkgs/*
 !aur-pkgs/build-aur-packages.sh
-!aur-pkgs/build-aur-package.sh
\ No newline at end of file
+!aur-pkgs/build-aur-package.sh
+rootfs/etc/makepkg.conf
```

Comment:
Don' t add unrelated changes. Split it up to other PR. Only the sed change should be made here

Reply:
This wasn't meant to end up in this merge request, this means something I did not intended ended up on this branch. I am revoking the merge request